### PR TITLE
DataFrame.add_prefix and  Series.add_prefix

### PR DESCRIPTION
**Pull request by yuwillrun**
Hello contributors .This is my first try to contribute to awesome koalas.
I begin to add some simple function to project. And i dont know how to add Codecov Report in pull requeset. Thank you for your guidance. 
Later I'll fix the add_prefix to serise.

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -2700,6 +2700,54 @@ def astype(self, dtype) -> 'DataFrame':
...
+        For DataFrame, the column labels are prefixed.
+        Parameters
+        ----------
+        prefix : str
+           The string to add before each label.
+        Returns
+        -------
+        Series or DataFrame
+           New Series or DataFrame with updated labels.
+        See Also
```
Comment (rxin) [inline databricks/koalas/frame.py]: Add a blank line here. When you copy on GitHub, it often drops blank lines ...

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -2700,6 +2700,54 @@ def astype(self, dtype) -> 'DataFrame':
...
+    def add_prefix(self, prefix):
+        """
+        Prefix labels with string `prefix`.
+        For Series, the row labels are prefixed.
+        For DataFrame, the column labels are prefixed.
+        Parameters
+        ----------
+        prefix : str
+           The string to add before each label.
+        Returns
```
Comment (rxin) [inline databricks/koalas/frame.py]: Blank line here too

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -2700,6 +2700,54 @@ def astype(self, dtype) -> 'DataFrame':
             self._metadata.index_columns + list(map(lambda ser: ser._scol, results)))
         return DataFrame(sdf, self._metadata.copy())
 
+    def add_prefix(self, prefix):
+        """
+        Prefix labels with string `prefix`.
+        For Series, the row labels are prefixed.
+        For DataFrame, the column labels are prefixed.
+        Parameters
```
Comment (rxin) [inline databricks/koalas/frame.py]: Blank line

File: databricks/koalas/frame.py [hunk 4]
```diff
@@ -2700,6 +2700,58 @@ def astype(self, dtype) -> 'DataFrame':
...
+        2  3  5
+        3  4  6
+        >>> df.add_prefix('col_')
+            col_A  col_B
+        0       1       3
+        1       2       4
+        2       3       5
+        3       4       6
+        """
+
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: remove an extra blank line.

**Comment by yuwillrun** [comment]
Add `Series.prefix` function

**Comment by yuwillrun** [comment]
emmm.. how to solve this checks failed  ?

**Comment by HyukjinKwon** [comment]
@yuwillrun, click the Travis CI results (the read X), see the error messages and fix it.

For instance, see https://travis-ci.com/databricks/koalas/jobs/204688677

**Pull request closed without merging**